priv/repo/migrations/20160808143454_create_category.exs

```diff
@@ -0,0 +1,15 @@
+defmodule CodeCorps.Repo.Migrations.CreateCategory do
+  use Ecto.Migration
+
+  def change do
+    create table(:categories) do
+      add :name, :string, null: false
+      add :slug, :string, null: false
+      add :description, :string
+
+      timestamps()
+    end
+
+    create index(:categories, [:slug], unique: true, name: :index_categories_on_slug)
```

Comment:
I didn't notice that in the schema / migrations in the ruby project, so I didn't add it here. Does ruby do that by default?

Also, I believe that if we create the index via `lower`, we'll have to make sure that whenever we query for the slug we always lower case it in the query. If that's correct I'll change it here :)